lib/manageiq/session/rack_session_dalli_logger.rb

```diff
@@ -0,0 +1,26 @@
+module ManageIQ
+  module Session
+    module RackSessionDalliLogger
+      def log_rack_session_access(*args, **_kwargs)
+        request = args.first
+        Rails.logger.debug("RackSessionDalliLogger##{caller_locations.first.label} id: #{request.request_id} method: #{request.request_method} fullpath: #{request.fullpath}")
```

Comment:
I'd like to align the values, and since the longest word is "delete_session", how about

```suggestion
        Rails.logger.debug("RackSessionDalliLogger##{caller_locations.first.label.ljust(14, ' ')} id: #{request.request_id} method: #{request.request_method} fullpath: #{request.fullpath}")
```

Reply:
Sure, fixed.  We'll still need to fix the logger since it screws it up based on the length of the thread id... 

```
    ...41.690606#92021:f4aec] DEBUG -- development: RackSessionDalliLogger#find_session   id: d5a25aa0-1b78-40c6-9c11-a07674599d12 method: GET fullpath: /dashboard/widget_report_data/24
    ...41.693611#92021:bc98] DEBUG -- development: RackSessionDalliLogger#find_session   id: ba0d3944-9fa2-432d-93ed-f224986a18cb method: GET fullpath: /dashboard/widget_chart_data/9
    ...41.713985#92021:f4a60] DEBUG -- development: RackSessionDalliLogger#find_session   id: 274972a4-07ab-4c08-b61f-719c586e86e5 method: GET fullpath: /dashboard/widget_report_data/22
```